src/main/java/org/folio/gobi/LookupService.java

```diff
@@ -62,28 +63,33 @@ public LookupService(RestClient restClient) {
     this.restClient = restClient;
   }
   /**
-   * Use the provided location code. if one isn't provided, or the specified
+   * Use the provided location code.
+   * If one isn't provided, or the specified
    * type can't be found, fallback using the first one listed
    *
-   * @param location
-   * @return UUID of the location
+   * @param locationCode code of location object
+   * @return UUID of the location object
    */
-  public CompletableFuture<String> lookupLocationId(String location) {
-    logger.debug("lookupLocationId:: Trying to look up locationId by location '{}'", location);
-    String query = HelperUtils.encodeValue(String.format(CQL_CODE_STRING_FMT, location));
-    String endpoint = String.format(LOCATIONS_ENDPOINT + QUERY, query);
-    return restClient.handleGetRequest(endpoint).toCompletionStage().toCompletableFuture()
+  public CompletableFuture<LocationTranslationResult> lookupLocationId(String locationCode) {
+    logger.debug("lookupLocationId:: Trying to look up locationId by location '{}'", locationCode);
+    return restClient.handleGetRequest(LOCATIONS_ENDPOINT).toCompletionStage().toCompletableFuture()
```

Comment:
please check if module permission is added to call mod-search enpoint

Reply:
Good catch, thanks. added permission and required module as well

Reply:
inventory-storage.locations.collection.get - and this one probably is not needed

Reply:
removed this one